debian/index.js

```diff
@@ -76,29 +77,24 @@ const start = () => {
         exec('kill -9 $(pgrep Headset) &> /dev/null')
       }
     })
+    
+    // Code based on MarshallOfSound Google-Play-Music-Desktop-Player-UNOFFICIAL-
+    // file src/main/features/linux/mediaKeysDBus.js
```

Comment:
it's good that you mention that. It' better to just provide the file to the link on github